File: EESoft/DataCenterVB/Backup/DataCenter - Client/Forms/MSGBOXFrm.vb
Public Class MSGBOXFrm
#Region "Form Auto Move Subroutain"
    Public x As Int16
    Public y As Int16
    Public fla As Boolean
    Private Sub MSGBOXFrm_MouseDown(ByVal sender As Object, ByVal e As System.Windows.Forms.MouseEventArgs) Handles MyBase.MouseDown, Panel.MouseDown
        x = e.X
        y = e.Y
        fla = True
    End Sub
    Private Sub MSGBOXFrm_MouseUp(ByVal sender As Object, ByVal e As System.Windows.Forms.MouseEventArgs) Handles MyBase.MouseUp, Panel.MouseUp
        fla = False
    End Sub
    Private Sub MSGBOXFrm_MouseMove(ByVal sender As Object, ByVal e As System.Windows.Forms.MouseEventArgs) Handles MyBase.MouseMove, Panel.MouseMove
        If fla = True Then
            Me.Location = New Point(Me.Location.X + (e.X - x), Me.Location.Y + (e.Y - y))
        Else
            Me.Visible = True
        End If

    End Sub
#End Region
    Private FadTMR As New Timer
    Private MSG As String
    Private LBLTMR As New Timer
    Public Sub New(ByVal ShowedMessage As String)
        MSG = ShowedMessage
        ' This call is required by the Windows Form Designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.

    End Sub
    Private Sub MSGBOXFrm_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        LBLMSG.Text = MSG
        FadTMR.Interval = 30
        AddHandler FadTMR.Tick, AddressOf FadTmr_tick
        AddHandler LBLTMR.Tick, AddressOf LBLTMR_Tick
        FadTMR.Enabled = True
        LBLTMR.Interval = 1500
        'LBLTMR.Enabled = True
    End Sub
    Private Sub FadTmr_tick(ByVal sender As Object, ByVal e As EventArgs)
        If Me.Opacity < 1 Then
            Me.Opacity += 0.029
        Else
            Me.Opacity = 1
            FadTMR.Enabled = False
        End If
    End Sub
    Private Sub LBLTMR_Tick(ByVal sender As Object, ByVal e As EventArgs)
        If LBLMSG.Visible = True Then
            LBLMSG.Visible = False
        Else
            LBLMSG.Visible = True
        End If
    End Sub
    Private Sub BtnOk_Click(ByVal sender As System.Object, ByVal e As System.EventArgs)
        LBLTMR.Enabled = False
        Me.Close()
    End Sub
End Class

File: EESoft/DataCenter/DataCenter - Server/Forms/MSGBOXFrm.cs
using System.Drawing;
using System.Windows.Forms;
using System;

namespace DataCenter_Server
{
	public partial class MSGBOXFrm
	{
		#region Form Auto Move Subroutain
		public short x;
		public short y;
		public bool fla;
        public void MSGBOXFrm_MouseDown(object sender, MouseEventArgs e)
        {
            x = (short)e.X;
            y = (short)e.Y;
            fla = true;
        }
        public void MSGBOXFrm_MouseUp(object sender, MouseEventArgs e)
        {
            fla = false;
        }
        public void MSGBOXFrm_MouseMove(object sender, MouseEventArgs e)
        {
            if (fla == true)
                Location = new Point(Location.X + (e.X - x), Location.Y + (e.Y - y));
            else
                Visible = true;

        }
		#endregion
		private Timer FadTMR = new Timer();
		private string MSG;
		private Timer LBLTMR = new Timer();
		public MSGBOXFrm(string ShowedMessage)
		{
			MSG = ShowedMessage;
			// This call is required by the Windows Form Designer.
			InitializeComponent();
			// Add any initialization after the InitializeComponent() call.
		}
        public void MSGBOXFrm_Load(Object sender, EventArgs e)
        {
            LBLMSG.Text = MSG;
            FadTMR.Interval = 30;
            FadTMR.Tick += FadTmr_tick;
            LBLTMR.Tick += LBLTMR_Tick;
            FadTMR.Enabled = true;
            LBLTMR.Interval = 1500;
            //LBLTMR.Enabled = True
        }
		private void FadTmr_tick(object sender, EventArgs e)
		{
			if (Opacity < 1)
				Opacity += 0.029;
			else
			{
				Opacity = 1;
				FadTMR.Enabled = false;
			}
		}
		private void LBLTMR_Tick(object sender, EventArgs e)
		{
			if (LBLMSG.Visible == true)
				LBLMSG.Visible = false;
			else
				LBLMSG.Visible = true;
		}
        private void BtnOk_Click(Object sender, EventArgs e)
        {
            LBLTMR.Enabled = false;
            Close();
        }
	}
}
